kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue

```diff
@@ -0,0 +1,309 @@
+<template>
+
+  <UiToolbar>
+    <KLabeledIcon :style="{ 'margin-top': '8px' }">
+      <template #icon>
+        <LearningActivityIcon
+          v-if="learningActivityKind"
+          data-test="learningActivityIcon"
+          :kind="learningActivityKind"
+          :shaded="true"
+        />
+      </template>
+      <TextTruncator
+        :text="resourceTitle"
+        :maxHeight="26"
```

Comment:
This magical number seems to work though I am not sure how exactly. Let me know if you see some possible issues with it or have more insight into if this is an appropriate value.